# This file contains example node parameters.
lane_detection_ai_node:
  ros__parameters:
    # Path of the model config
    model_config_path: "config/model_sparse_config.py"

    # Topics for an example publisher and subscriber
    image_topic: "/camera/image/undistorted"
    result_topic: "/lane_detection/lane"
    debug_image_topic: "/lane_detection/debug/image"
